easybuild/easyconfigs/b/bpp-core/bpp-core-2.4.1-GCC-8.2.0-2.31.1.eb

```diff
@@ -0,0 +1,24 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+easyblock = 'CMakeMake'
+
+name = 'bpp-core'
+version = '2.4.1'
+
+homepage = 'https://github.com/BioPP/bpp-core'
+description = """Bio++ Core Library"""
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
+
+github_account = 'BioPP'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['1150b8ced22cff23dd4770d7c23fad11239070b44007740e77407f0d746c0af6']
+
+builddependencies = [('CMake', '3.13.3')]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ["lib64"]
```

Comment:
@omula I would make this a bit more specific:

```python
sanity_check_paths = {
    'files': ['lib64/libbpp-core.a', 'lib64/libbpp-core.%s' % SHLIB_EXT],
    'dirs': ['include/Bpp']
}

Reply:
done